### ISSUE TITLE
Support specifying plot title

### ISSUE THREAD

**Issue by adworacz** [Zoxc/crusader#12]
Thank you for this awesome tool!

Would it be possible to support specifying a plot title? Primary use case being to easily differentiate between changes in wireless or router settings between test runs. Sure we can use different file names, but if we upload results to something like the OpenWRT forums, the filename is effectively lost and the exact settings used aren't communicated on the actual graph.

This feature would match flent's current capability as well.

**Comment by mcuee** [comment]
Indeed this can be a useful feature.

**Comment by Zoxc** [comment]
I have added this option to the CLI at least.

**Comment by mcuee** [comment]
> I have added this option to the CLI at least.

Yes, that is correct. The GUI version does not seem to have this option.

**Comment by mcuee** [comment]
@Zoxc 

I think this issue can be closed now.
